tests/console/kdump_and_crash.pm

```diff
@@ -27,15 +27,13 @@ sub run() {
     # restart to activate kdump
     script_run 'reboot', 0;
     $self->wait_boot;
-    reset_consoles;
     select_console 'root-console';
 
     # often kdump could not be enabled: bsc#1022064
     return 1 unless kdump_is_active;
     do_kdump;
     # wait for system's reboot
     $self->wait_boot;
-    reset_consoles;
```

Comment:
the reset_consoles *after* wait_boot are actually quite dangerous, because wait_boot is quite a function selecting consoles